lib/components/map/leg-diagram.js

```diff
@@ -289,7 +290,7 @@ class LegDiagram extends Component {
           const stepWidth = stepArr[i + 1] - beginStep
           backgroundSvgContent.push(
             <rect
-              fill="#eee"
+              fill={grey[50]}
```

Comment:
I'm hesitant to define these color variables by how saturated they are, since the `grey` variable is a spectrum. `grey[50]` is dim, but  so is `grey[100]`